nncf/tensorflow/graph/patterns.py

```diff
@@ -98,3 +98,23 @@ def create_h_swish_act() -> GraphPattern:
     main_pattern.add_pattern_alternative(pattern)
 
     return main_pattern
+
+
+def create_matmul_biasadd_pattern() -> GraphPattern:
```

Comment:
Do `create_matmul_biasadd_pattern` and `create_conv2d_biasadd_pattern` support Keras Layers?

Reply:
No, it doesn't. The reason for these changes is following:

If we pass `use_bias=True` (this is the default value) to the `__init__()` method of the `tf.keras.layers.Conv2D` or `tf.keras.layers.Dense`  layer we will get two operations `Conv2D + BiasAdd` or `MatMul + BiasAdd` respectively. These operations will be inside TF graph (two nodes in the NNCF graph). But we have only one operation inside the Keras graph (one node in the NNCF graph).